javers-core/src/main/java/org/javers/common/collections/Optional.java

```diff
@@ -2,10 +2,14 @@
 
 import org.javers.common.validation.Validate;
 
+import java.io.Serializable;
+
 /**
  * @author bartosz walacik
  */
-public class Optional<T> {
+public class Optional<T> implements Serializable{
+    private static final long serialVersionUID = 72109946687664234L;
```

Comment:
dodawanie z palca serialVersionUID jest kontrowersyjne, jeśli ktoś później zmieni tą klasę a nie zmieni
serialVersionUID to może doprowadzić do bardzo brzydkich błędów w runtime, proponuję usunąć serialVersionUID, kompilator wygeneruje